src/components/UI/sidepanels/MainSidepanel/MainSidepanel.styles.ts

```diff
@@ -1,11 +1,11 @@
 import styled from 'styled-components'
 
 export const Wrapper = styled.div`
-  width: 208px;
-  height: 100vh;
-  position: fixed;
+  /* width: 208px; */
```

Comment:
for what?

Reply:
There is a special component in antd library for that - Sider. There is no need to create a custom one on our own

Reply:
so just delete it